models/bamf_mr_brain_tumor/meta.json

```diff
@@ -0,0 +1,206 @@
+{
+  "id": "",
+  "name": "bamf_mr_brain_tumor",
+  "title": "BAMF MR Brain Tumor",
+  "summary": {
+    "description": "An nnU-Net based model to segment brain tumors from CT scans",
```

Comment:
This model has a lot under the hood and generates several output files. It'd be nice to give it a bit more detail in the  desctiption.

Reply:
I have updated this. Please let me know if more info needed.